schematypes.go

```diff
@@ -20,12 +20,33 @@ package dap
 
 // Message is an interface that all DAP message types implement with pointer
 // receivers. It's not part of the protocol but is used to enforce static
-// typing in Go code.
+// typing in Go code and provide some common accessors.
 //
 // Note: the DAP type "Message" (which is used in the body of ErrorResponse)
 // is renamed to ErrorMessage to avoid collision with this interface.
 type Message interface {
-	isMessage()
+	GetSeq() int
```

Comment:
According to [Effective Go](https://golang.org/doc/effective_go.html#Getters) we should be calling these Seq().

Reply:
Correct. However, our structs already have a field named `Seq` and thus can't have a `Seq` method. Since we already have a lot of code using the `Seq` field and it's part of the DAP protocol, I opted for keeping it as is. Same for `Type`.